include/splinterdb/splinterdb.h

```diff
@@ -179,7 +179,7 @@ splinterdb_open(splinterdb_config *cfg, splinterdb **kvs);
 // Close a splinterdb
 //
 // This will flush all data to disk and release all resources
-void
+int
```

Comment:
To percolate errors found by shm-destroy, if large-fragments not free are still found hanging around.